src/test/41-redis.ts

```diff
@@ -0,0 +1,1407 @@
+      });
+      it("redis.scan", async () => {
+        const response = await c8Client.redis.scan(0, collectionName);
+        expect(response.code).to.equal(200);
+      });
+      it("redis.scan2", async () => {
+        const response = await c8Client.redis.scan(0, collectionName, "*", 100);
+        expect(response.code).to.equal(200);
+      });
+      it("redis.ttl", async () => {
+        await c8Client.redis.set("ttl", "test", collectionName);
+        const response = await c8Client.redis.ttl("ttl", collectionName);
+        expect(response.result).to.equal(-1);
+        expect(response.code).to.equal(200);
+      });
+      it("redis.type", async () => {
+        await c8Client.redis.set("type", "test", collectionName);
+        const response = await c8Client.redis.type("type", collectionName);
+        expect(response.result).to.equal("string");
+        expect(response.code).to.equal(200);
+      });
+      it("redis.unlink", async () => {
+        await c8Client.redis.set("unlink1", "test", collectionName);
+        await c8Client.redis.set("unlink2", "test", collectionName);
+        const response = await c8Client.redis.unlink(
+          ["unlink1", "unlink2"],
+          collectionName
+        );
+        expect(response.result).to.equal(2);
+        expect(response.code).to.equal(200);
+      });
+    });
+    describe("test redis server commands", () => {
+      it("redis.echo", async () => {
+        const response = await c8Client.redis.echo(
+          "Hello World!",
+          collectionName
+        );
+        expect(response.result).to.equal("Hello World!");
+        expect(response.code).to.equal(200);
+      });
+      it("redis.ping", async () => {
+        const response = await c8Client.redis.ping(collectionName);
+        expect(response.result).to.equal("PONG");
+        expect(response.code).to.equal(200);
+      });
+      it("redis.ping2", async () => {
+        const response = await c8Client.redis.ping(
+          collectionName,
+          "Hello World!"
+        );
+        expect(response.result).to.equal("Hello World!");
+        expect(response.code).to.equal(200);
+      });
+      it("redis.dbsize", async () => {
+        const response = await c8Client.redis.dbsize(collectionName);
+        expect(response.code).to.equal(200);
+      });
+      it("redis.flushdb", async () => {
+        const response = await c8Client.redis.flushdb(collectionName);
+        expect(response.result).to.equal("OK");
+        expect(response.code).to.equal(200);
+      });
+      it("redis.flushdb2", async () => {
+        const response = await c8Client.redis.flushdb(collectionName, true);
+        expect(response.result).to.equal("OK");
+        expect(response.code).to.equal(200);
+      });
+      it("redis.time", async () => {
+        const response = await c8Client.redis.time(collectionName);
+        expect(response.code).to.equal(200);
+      });
+    });
+  });
+});
```

Comment:
Please validate negative scenarios as well.

Reply:
Agreed! I will add a couple of bad requests.